notebooks/ASC March Acceptance Test Notebook.ipynb

```diff
@@ -2841,7 +2841,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
```

Comment:
Doesn't look like there are any substantial changes in this notebook. Consider trying to reset it.